ports/qemu-riscv/README.md

```diff
@@ -0,0 +1,38 @@
+This is experimental, community-supported port for RISC-V RV32IM emulation as
+provided by QEMU (http://qemu.org).
+
+The purposes of this port are to enable:
+
+1. Continuous integration
+    - run tests against architecture-specific parts of code base
+2. Experimentation
+    - simulation & prototyping of anything that has architecture-specific
+      code
+    - exploring instruction set in terms of optimising some part of
+      MicroPython or a module
+3. Streamlined debugging
+    - no need for JTAG or even an MCU chip itself
+    - no need to use OpenOCD or anything else that might slow down the
+      process in terms of plugging things together, pressing buttons, etc.
+
+This port requires a newlib based bare metal/ELF RISC-V toolchain, either
+with multilib support or 32 bits specific (M, C, and Zicsr extensions must
+be supported, along with ilp32 ABI).  Either musl-based or glibc-based
+toolchains are not tested or known to work.
```

Comment:
This last sentence is confusing: have these toolchains been tested?  Maybe just remove this sentence altogether.

Reply:
Will do, thanks for noticing.